### ISSUE TITLE
Let's Encrypt ./run: line 282:   401 Illegal instruction     (core dumped)

### ISSUE THREAD

**Issue by SCRAME314** [home-assistant/addons#3591]
### Describe the issue you are experiencing

Can't update cert after HA update
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
cont-init: info: running /etc/cont-init.d/file-structure.sh
cont-init: info: /etc/cont-init.d/file-structure.sh exited 0
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
services-up: info: copying legacy longrun lets-encrypt (no readiness notification)
s6-rc: info: service legacy-services successfully started
**[21:01:43] INFO: Selected http verification**
[21:01:44] INFO: **Detecting existing certificate type for _xxxx.xxxx.xx_**
./run: line 282:   402 Illegal instruction     (core dumped) certbot certonly --non-interactive --keep-until-expiring --expand --email "$EMAIL" --agree-tos "${KEY_ARGUMENTS[@]}" --cert-name "${DOMAIN_ARR[1]}" "${DOMAIN_ARR[@]}" --config-dir "$CERT_DIR" --work-dir "$WORK_DIR" --preferred-challenges "$CHALLENGE" "${ACME_CUSTOM_SERVER_ARGUMENTS[@]}" --standalone --preferred-chain "ISRG Root X1"
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped

### What type of installation are you running?

Home Assistant Supervised

### Which operating system are you running on?

Debian

### Which add-on are you reporting an issue with?

Let's Encrypt

### What is the version of the add-on?

 5.0.18 

### Steps to reproduce the issue

1. amd64 nettop
2. Debian Linux 6.1.0-18-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.76-1 (2024-02-01) x86                                                                             _64
3. installed HA 2024.03.XX
4. Reverted Backup from ARMbased device
5. Cert was till 06.05.2024
6. Installed update of Let's Encrypt addon & HA CORE update
7. Tried to start Let'sEncrypt addon to update cert today, but got an error

### System Health information

Core
2024.5.2
Supervisor
2024.04.4
Operating System
Пользовательский интерфейс
20240501.1

### Anything in the Supervisor logs that might be useful for us?

```txt
2024-05-07 20:50:29.290 INFO (MainThread) [supervisor.homeassistant.core] Home Assistant Core state changed to NOT_RUNNING
2024-05-07 20:50:31.216 INFO (MainThread) [supervisor.auth] Auth request from 'core_mosquitto' for 'homeassistant'
2024-05-07 20:50:31.239 INFO (MainThread) [supervisor.auth] Home Assistant not running, checking cache
2024-05-07 20:50:44.438 INFO (MainThread) [supervisor.homeassistant.core] Home Assistant Core state changed to STARTING
2024-05-07 20:50:50.026 INFO (MainThread) [supervisor.homeassistant.core] Home Assistant Core state changed to RUNNING
2024-05-07 20:50:50.026 INFO (MainThread) [supervisor.homeassistant.core] Detect a running Home Assistant instance
2024-05-07 20:53:41.271 INFO (MainThread) [supervisor.addons.addon] No core_letsencrypt addon Docker image homeassistant/amd64-addon-letsencrypt found
2024-05-07 20:53:41.272 INFO (MainThread) [supervisor.docker.interface] Downloading docker image homeassistant/amd64-addon-letsencrypt with tag 5.0.18.
2024-05-07 20:55:26.029 WARNING (MainThread) [supervisor.addons.options] Unknown option 'base_topic' for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 20:55:26.030 WARNING (MainThread) [supervisor.addons.options] Option 'advanced' does not exist in the schema for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 20:55:26.037 WARNING (MainThread) [supervisor.addons.options] Unknown option 'base_topic' for Zigbee2MQTT 2 (21a8b9e4_zigbee2mqtt)
2024-05-07 20:57:07.863 WARNING (MainThread) [supervisor.addons.options] Unknown option 'base_topic' for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 20:57:07.863 WARNING (MainThread) [supervisor.addons.options] Option 'advanced' does not exist in the schema for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 20:57:26.851 INFO (MainThread) [supervisor.addons.addon] Creating Home Assistant add-on data folder /data/addons/data/core_letsencrypt
2024-05-07 20:57:26.857 INFO (MainThread) [supervisor.docker.interface] Downloading docker image homeassistant/amd64-addon-letsencrypt with tag 5.0.18.
2024-05-07 20:57:35.772 INFO (MainThread) [supervisor.addons.manager] Add-on 'core_letsencrypt' successfully installed
2024-05-07 20:57:37.607 WARNING (MainThread) [supervisor.addons.manager] Add-on core_letsencrypt is already installed
2024-05-07 21:00:26.027 WARNING (MainThread) [supervisor.addons.options] Unknown option 'base_topic' for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 21:00:26.028 WARNING (MainThread) [supervisor.addons.options] Option 'advanced' does not exist in the schema for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 21:00:26.036 WARNING (MainThread) [supervisor.addons.options] Unknown option 'base_topic' for Zigbee2MQTT 2 (21a8b9e4_zigbee2mqtt)
2024-05-07 21:00:26.113 WARNING (MainThread) [supervisor.addons.options] Unknown option 'base_topic' for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 21:00:26.114 WARNING (MainThread) [supervisor.addons.options] Option 'advanced' does not exist in the schema for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 21:00:26.121 WARNING (MainThread) [supervisor.addons.options] Unknown option 'base_topic' for Zigbee2MQTT 2 (21a8b9e4_zigbee2mqtt)
2024-05-07 21:01:24.874 INFO (MainThread) [supervisor.docker.addon] Starting Docker add-on homeassistant/amd64-addon-letsencrypt with version 5.0.18
2024-05-07 21:05:26.031 WARNING (MainThread) [supervisor.addons.options] Unknown option 'base_topic' for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 21:05:26.032 WARNING (MainThread) [supervisor.addons.options] Option 'advanced' does not exist in the schema for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 21:05:26.041 WARNING (MainThread) [supervisor.addons.options] Unknown option 'base_topic' for Zigbee2MQTT 2 (21a8b9e4_zigbee2mqtt)
2024-05-07 21:10:26.031 WARNING (MainThread) [supervisor.addons.options] Unknown option 'base_topic' for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 21:10:26.032 WARNING (MainThread) [supervisor.addons.options] Option 'advanced' does not exist in the schema for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 21:10:26.040 WARNING (MainThread) [supervisor.addons.options] Unknown option 'base_topic' for Zigbee2MQTT 2 (21a8b9e4_zigbee2mqtt)
2024-05-07 21:11:33.103 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
2024-05-07 21:15:26.030 WARNING (MainThread) [supervisor.addons.options] Unknown option 'base_topic' for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 21:15:26.031 WARNING (MainThread) [supervisor.addons.options] Option 'advanced' does not exist in the schema for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 21:15:26.040 WARNING (MainThread) [supervisor.addons.options] Unknown option 'base_topic' for Zigbee2MQTT 2 (21a8b9e4_zigbee2mqtt)
2024-05-07 21:20:26.032 WARNING (MainThread) [supervisor.addons.options] Unknown option 'base_topic' for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 21:20:26.033 WARNING (MainThread) [supervisor.addons.options] Option 'advanced' does not exist in the schema for Zigbee2MQTT (45df7312_zigbee2mqtt)
2024-05-07 21:20:26.042 WARNING (MainThread) [supervisor.addons.options] Unknown option 'base_topic' for Zigbee2MQTT 2 (21a8b9e4_zigbee2mqtt)
```

**Comment by deniswal** [comment]
Idem for me:

**Comment by YSmetana** [comment]
>  Apparently the new certbot version uses an instruction not recognized by "old" CPUs.

Thank you @deniswal ! Had rolled back to the backup version  5.0.1 of the addon and certificated updated!

**Comment by SCRAME314** [comment]
unfortunately I can not roll back to backup, because I have deleted it. So could I install previous version of this addon somehow?

**Comment by SCRAME314** [comment]
Have found full backup 13.04.24 & restored only Let'sEncrypt addon version 5.0.15. The cert has been successfully renewed after downgrade Let's Encrypt addon version, so the issue is in 5.0.18

**Comment by dimfishr** [comment]
HOWTO Rollback to 5.0.15 without old backups:
1. Create new partial backup for Let's Encrypt only.
2. Download and extract backup tar file.
3. Replace in 'core_letsencrypt.tar.gz/addon.json' all version text 5.0.18 => 5.0.15 
4. Replace new file core_letsencrypt.tar.gz in backup tar (don't change backup.json)
5. Upload backup to HA and restore.

**Comment by tomeralgai** [comment]
Has anyone found a solution for this other than rolling back?

**Comment by tomeralgai** [comment]
Does anyone know if this issue has been resolved in Let's Encrypt V 5.0.23 or should I stay on 5.0.15?

**Comment by peterwesterlund** [comment]
> HOWTO Rollback to 5.0.15 without old backups:
> 
> 1. Create new partial backup for Let's Encrypt only.
> 2. Download and extract backup tar file.
> 3. Replace in 'core_letsencrypt.tar.gz/addon.json' all version text 5.0.18 => 5.0.15
> 4. Replace new file core_letsencrypt.tar.gz in backup tar (don't change backup.json)
> 5. Upload backup to HA and restore.

I can't manage to re-create that. Getting:

```
2024-11-19 04:06:17.762 ERROR (MainThread) [supervisor.backups.backup] Can't find backup core_letsencrypt
2024-11-19 04:06:17.764 WARNING (MainThread) [supervisor.backups.backup] Can't restore Add-on core_letsencrypt: Can't find backup core_letsencrypt
```
